### PR TITLE
kitty: update to 0.11.2

### PR DESCRIPTION
**Pull request by emacsomancer**

### CODE REVIEW

File: srcpkgs/kitty/template [hunk 1]
```diff
@@ -9,11 +9,11 @@ makedepends="gettext-devel glfw-devel harfbuzz-devel libxkbcommon-devel
 depends="kitty-terminfo"
 conf_files="/etc/kitty/kitty.conf"
 short_desc="Modern, hackable, featureful, OpenGL based terminal emulator"
-maintainer="Julio Galvan <juliogalvan@protonmail.com>"
+maintainer="Benjamin Slade <slade@jnanam.net>"
```
Comment (cr6git) [inline srcpkgs/kitty/template]: Why you changed the maintainer ?

Comment (emacsomancer) [reply srcpkgs/kitty/template]: It's certainly not critical. I was just signalling willingness to be responsible for this one.

Comment (cr6git) [reply srcpkgs/kitty/template]: ping @julio641742

Comment (cr6git) [reply srcpkgs/kitty/template]: @emacsomancer the commit message must be: `kitty: update to 0.11.2.`

Comment (julio641742) [reply srcpkgs/kitty/template]: I don't mind if you commit this change @emacsomancer @cr6git, in my defense I have been in vacations this past month so I have not had a chance to update the packages I maintain yet.

**Pull request merged**